dev/test/dev/debug_qp_test.clj

```diff
@@ -28,56 +28,55 @@
               :fk-field-id  (mt/id :venues :category_id)}])))))
 
 (deftest to-mbql-shorthand-test
-  (mt/dataset sample-dataset
-    (testing "Normal Field ID clause"
-      (is (= '$user_id
-             (debug-qp/expand-symbolize [:field (mt/id :orders :user_id) nil])))
-      (is (= '$products.id
-             (debug-qp/expand-symbolize [:field (mt/id :products :id) nil]))))
```

Comment:
Most of this PR's diff is whitespace-only, from removing `(mt/dataset sample-dataset` and unindenting the lines it wrapped.

You're probably well aware of this, but it helped me to look at this with [whitespace disabled](https://github.com/metabase/metabase/pull/35973/files?diff=unified&w=1).